docs/architecture/sstable/sstable2/sstable-interpretation.rst

```diff
@@ -22,7 +22,7 @@ is just a **mutation**, a list of changed (added or deleted) columns and
 their new values (or "tombstone" for a deleted column), and a timestamp
 for each such change (this timestamp is used for reconciling conflicting
 mutations). The full data row needed by a request will be composed from
-potentially multiple sstables and/or the in-memory table(s).
+potentially multiple sstables.
```

Comment:
This change is wrong :-( This document talks about sstables and memtables - not the "in memory tables" that your patch is about.

Reply:
Yeah, you're right.
OTOH memtables are irrelevant in the context of this paragraph, so the change isn't that bad either.
For completeness, the process of integrating mutations from multiple readers, including SSTables, memtable, and cache, should be documented elsewhere.